src/scenes/ApplicationRouter/portal/Prescreen/index.tsx

```diff
@@ -43,10 +47,28 @@ const PrescreenView = () => {
   }
 };
 
-const PrescreenViewWrapper = () => (
-  <ApplicationPage>
-    <PrescreenView />
-  </ApplicationPage>
-);
+const PrescreenViewWrapper = () => {
+  const { activeLocale } = useLocalization();
+  const { selectedApplication } = useApplicationData();
+
+  const crumbs: Crumb[] = useMemo(
+    () =>
+      activeLocale && selectedApplication?.id
+        ? [
+            {
+              name: strings.ALL_SECTIONS,
+              to: APP_PATHS.APPLICATION_OVERVIEW.replace(':applicationId', `${selectedApplication.id}`),
+            },
+          ]
+        : [],
+    [activeLocale, selectedApplication?.id]
+  );
+
+  return (
+    <ApplicationPage crumbs={crumbs}>
+      <PrescreenView />
+    </ApplicationPage>
+  );
+};
```

Comment:
Wasn't sure if the intention was to configure the breadcrumbs for all of the views in this PR, but started with adding breadcrumbs to the Prescreen views to confirm this aligns with your expectations @tommylau523.

Reply:
Sounds good to me! I think since not all the pages are built, we can focused on the ones that are easily testable. It should be relatively easy to add the crumbs as we build them page by page.